content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -52,6 +54,148 @@ This command does two things:
 
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'mysql']}>
+
+In this guide, you'll use [Prisma Migrate](/concepts/components/prisma-migrate) to create the tables in your database. Add the following Prisma data model to your Prisma schema in `prisma/schema.prisma`:
+
+```prisma file=prisma/schema.prisma copy
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  title     String   @db.VarChar(255)
+  content   String?
+  published Boolean  @default(false)
+  author    User     @relation(fields: [authorId], references: [id])
+  authorId  Int
+}
+
+model Profile {
+  id     Int     @id @default(autoincrement())
+  bio    String?
+  user   User    @relation(fields: [userId], references: [id])
+  userId Int     @unique
+}
+
+model User {
+  id      Int      @id @default(autoincrement())
+  email   String   @unique
+  name    String?
+  posts   Post[]
+  profile Profile?
+}
+```
+
+To map your data model to the database schema, you need to use the `prisma migrate` CLI commands:
+
+```terminal
+npx prisma migrate dev --name init
+```
+
+This command does two things:
+
+1. It creates a new SQL migration file for this migration
+1. It runs the SQL migration file against the database
+
+> **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'planetscale']}>
+
+In this guide, you'll use Prisma's [`db push` command](/concepts/components/prisma-migrate/db-push) to create the tables in your database. Add the following Prisma data model to your Prisma schema in `prisma/schema.prisma`:
+
+```prisma file=prisma/schema.prisma copy
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  title     String   @db.VarChar(255)
+  content   String?
+  published Boolean  @default(false)
+  author    User     @relation(fields: [authorId], references: [id])
+  authorId  Int
+
+  @@index(authorId)
+}
+
+model Profile {
+  id     Int     @id @default(autoincrement())
+  bio    String?
+  user   User    @relation(fields: [userId], references: [id])
+  userId Int     @unique
+
+  @@index(userId)
+}
+
+model User {
+  id      Int      @id @default(autoincrement())
+  email   String   @unique
+  name    String?
+  posts   Post[]
+  profile Profile?
+}
+```
+
+You are now ready to push your new schema to your database. Connect to your `main` branch using the instructions in [Connect your database](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-planetscale/).
```

Comment:
This link is currently 404 as the page is not depoyled yet, that was the one that otherwise triggered a link checker error, correct? 

If so:
```suggestion
You are now ready to push your new schema to your database. Connect to your `main` branch using the instructions in [Connect your database](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-planetscale/)<!-- Absolute external link as otherwise the link checker reports this link as broken even it should be valid -->.
```

Reply:
Correct. Will add your suggested comment.